client/src/store/comments/reducers.ts

```diff
@@ -51,24 +59,33 @@ export const commentsReducer = (state: IComments = defaultState, action: { type:
                 commentsLoading: true,
             };
         case GET_COMMENTS_SUCCESS:
+            const {normalizedComments: {entities: {authors, comments}, result}, postId, page}:
```

Comment:
where do you use authors, comments ?

Reply:
return {
                ...state,
                comments: {...state.comments, ...comments},
                authors: {...state.authors, ...authors},
                allCommentsId: [...state.allCommentsId, ...result],
                commentsPage: commentsState,
                commentsLoading: false,
            };